geode-core/src/main/java/org/apache/geode/management/internal/cli/shell/Gfsh.java

```diff
@@ -14,38 +14,10 @@
  */
```

Comment:
How come the format of the file changes so much. The order gets switched a lot. Some comments get moved onto another method. Is this happening because of spotless or your editor?